### PR TITLE
Add release issue template

### PR DESCRIPTION
**Pull request by maarcingebala**
This template will be used to create issues where we'll be tracking progress on new minor releases of Saleor.

### CODE REVIEW

**Review by andrzejewsky — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/release.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- [ ] Check the change log and do smoke tests - @michalina-graczyk
+- [ ] Inform the team about results of the tests - @michalina-graczyk
+- [ ] Perform smoke check on demo and storefront - @michalina-graczyk
+- [ ] Inform about finished tests - @michalina-graczyk
+- [ ] Release stable tag - @maarcingebala
+- [ ] Release stable tag  - @andrzejewsky
+- [ ] Accepts release PR to sandbox - @michalina-graczyk
+- [ ] Inform about approved release PR to sandbox  - @michalina-graczyk
+- [ ] Merge release PR to sandbox - @cmiacz
+- [ ] Inform about stable release (sandbox) - @czarek
```
Comment (patrys) [inline .github/ISSUE_TEMPLATE/release.md:30]: Which handle is correct?

Comment (maarcingebala) [reply .github/ISSUE_TEMPLATE/release.md]: fixed

**Comment by korycins** [comment]
It is only a detail but Is it okay that you rely on a specific person? What if you will be off or something? Shouldn't we have here assigned groups? Even if this will be single person group.

**Comment by maarcingebala** [comment]
@korycins We don't want to ping entire teams, only responsible people. If they'll be off/unavailable, we'll agree on who's replacing them and change them in the task.

File: .github/ISSUE_TEMPLATE/release.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+This issue tracks the release progress of Saleor <X.Y.Z>.
+
+## Steps
+
+- [ ] Release alpha tag - @maarcingebala
+- [ ] Inform about alpha tag - @maarcingebala
+- [ ] Release alpha tag - @andrzejewsky
+- [ ] inform about alpha tag - @andrzejewsky
```
Comment (IKarbowiak) [inline .github/ISSUE_TEMPLATE/release.md:16]: 🚔 
```suggestion
- [ ] Inform about alpha tag - @andrzejewsky
```

File: .github/ISSUE_TEMPLATE/release.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+This issue tracks the release progress of Saleor <X.Y.Z>.
+
+## Steps
+
+- [ ] Release alpha tag - @maarcingebala
+- [ ] Inform about alpha tag - @maarcingebala
+- [ ] Release alpha tag - @andrzejewsky
+- [ ] Inform about alpha tag - @andrzejewsky
+- [ ] Prepare environments for the new minor and trigger deployments - @cmiacz
+- [ ] Inform about finished deployment on slack channel - @cmiacz
```
Comment (andrzejewsky) [inline .github/ISSUE_TEMPLATE/release.md:18]:
```suggestion
- [ ] Inform about finished deployment - @cmiacz
```

**Review by andrzejewsky — CHANGES_REQUESTED** [review]

**Review by andrzejewsky — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Pull request merged**